_posts/1-2-28-InlineGallery.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: doc
+permalink: /docs/ui-toolkit/components/inline-gallery
+title: Inline Gallery
+section: UI toolkit
+---
+
+# InlineGallery
+
+InlineGallery component renders a collection of `Image` components within `HorizontalPager`. Each preview component is rendered on a separate page.
```

Comment:
Based on image and description, I don't understand what's the difference between `ImageGallery` and `InlineGallery`.